src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -401,6 +402,7 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 -->
 <!-- As we split up/refactor this file, put new imports here -->
 <Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.D8.targets" />
+<Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.DX.targets" />
```

Comment:
is there a way to conditionally import this I wonder. So only if `AndroidLinkTool == dx`.

Reply:
I'll try it, I think I will just have to name the target the same in both files.

Reply:
what we can do is add an extension point and then use that in the new Dx.targets.
it should work. the only call we have to Aapt/Aapt2 now I believe will be the CallTarget calls. 
Although I just thought of a way to work around that now I think...